### DISCUSSION TITLE
Possible to make clipboard history enabled?

### DISCUSSION THREAD

**Discussion by lukaspavelka** [LeDragoX/Win-Debloat-Tools#144] [Q&A] [answered]
Hello, 

please advice how to make clipboard history enabled

when i run tweaks as : CLI Version (Minimal, good for automation) ?

Is there any config or some easy way how to change settings? 
Sorry I'm not very familiar with PowerShell scripts.

**Comment by LeDragoX** [comment, marked as answer]
Hello, yes and sorry for the late response, you can use the GUI version and on the left corner of the UI you'll find an option named `"Enable Clipboard History"`, just tick one time to enable (again if you want to disable).
```ps1
Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force; ls -Recurse *.ps*1 | Unblock-File; .\"WinDebloatTools.ps1"
```

Secondly, after running the _CLI/GUI_, if you still have the **Terminal** opened, just type `Enable-ClipboardHistory` and hit **ENTER**, because the functions are kept imported after executing the script one time, but closing the **Terminal** resets this.

If you want more commands, you can find them here: `src\utils\Individual-Tweaks.psm1`
